### PR TITLE
Zoo update

### PR DESCRIPTION
**Pull request by koenhelwegen**
Update docs with recent changes to `zoo` (to be merged after next `zoo` release):
- literature/sota split
- add quicknet to table

### CODE REVIEW

**Comment by koenhelwegen** [comment]
@lgeiger the links to github here are broken, it points to https://github.com/larq/zoo/blob/v1.0b2/larq_zoo/sota/quicknet.py#L131 while it should be [https://github.com/larq/zoo/blob/**v1.0.b2**/larq_zoo/sota/quicknet.py#L131](https://github.com/larq/zoo/blob/v1.0.b2/larq_zoo/sota/quicknet.py#L131), any idea how to fix this?

**Comment by lgeiger** [comment]
> the links to github here are broken

Not sure, the links are generated [here](https://github.com/larq/docs/blob/master/generate_api_docs.py#L40-L42).

File: generate_api_docs.py [hunk 1]
```diff
@@ -38,8 +38,10 @@ def callable_to_source_link(obj):
     source = inspect.getsourcelines(obj)
     line = source[-1] + 1 if source[0][0].startswith("@") else source[-1]
     version = get_version(package_name)
+    version = version.replace("b", ".b")  # correct github links to beta releases
```
Comment (koenhelwegen) [inline generate_api_docs.py]: This is not very pretty, would be glad to replace it with a less hacky solution

Comment (koenhelwegen) [reply generate_api_docs.py]: One way to make this more robust would be to check if the link is valid via `urllib` as we run this script. @lgeiger how do you feel about this?

Comment (lgeiger) [reply generate_api_docs.py]: That's fine, looks like PyPI names the release [1.0b2](https://pypi.org/project/larq-zoo/1.0b2/) which is different to the git tag

File: docs/zoo/index.md [hunk 2]
```diff
@@ -5,32 +5,50 @@ These models can be used for prediction, feature extraction, and fine-tuning.
...
+Larq Zoo consists of a `literature` and a `sota` submodule.
+
+The `literature` submodule contains replications from research papers (all current models).
+These models are intended to provide a stable reference for ideas presented in specific papers.
+The model implementations will be maintained, but we will not attempt to improve these models over time by applying new training strategies or architecture innovations.
+
+The `sota` submodule contains top models for various scenarios. These models are intended to use in a [`SW 2.0`](https://medium.com/@karpathy/software-2-0-a64152b37c35)-like fashion.
+We will do our best to continuously improve the models, meaning their weights and even details about there architecture may change from release to release.
```
Comment (timdebruin) [inline docs/zoo/index.md]:
```suggestion
We will do our best to continuously improve the models, which means that their weights and even details of their architectures may change from release to release.
```

File: docs/zoo/index.md [hunk 3]
```diff
@@ -5,32 +5,50 @@ These models can be used for prediction, feature extraction, and fine-tuning.
...
+| [BinaryDenseNet37Dilated](/zoo/api/literature/#binarydensenet37dilated) | 64.34 %        | 85.15 %        | 8 734 120  | 5.25 MB  |
+| [BinaryDenseNet37](/zoo/api/literature/#binarydensenet37)               | 62.89 %        | 84.19 %        | 8 734 120  | 5.25 MB  |
+| [BinaryDenseNet28](/zoo/api/literature/#binarydensenet28)               | 60.91 %        | 82.83 %        | 5 150 504  | 4.12 MB  |
+| [BinaryResNetE18](/zoo/api/literature/#binaryresnete18)                 | 58.32 %        | 80.79 %        | 11 699 368 | 4.03 MB  |
+| [Bi-Real Net](/zoo/api/literature/#birealnet)                           | 57.47 %        | 79.84 %        | 11 699 112 | 4.03 MB  |
+| [DoReFaNet](/zoo/api/literature/#dorefanet)                             | 53.39 %        | 76.50 %        | 62 403 912 | 22.84 MB |
+| [XNOR-Net](/zoo/api/literature/#xnornet)                                | 44.96 %        | 69.18 %        | 62 396 768 | 22.81 MB |
+| [Binary AlexNet](/zoo/api/literature/#binaryalexnet)                    | 36.30 %        | 61.53 %        | 61 859 192 | 7.49 MB  |
+
+The `sota` submodule contains these models:
```
Comment (lgeiger) [inline docs/zoo/index.md]: I think we should link the the API docs here

File: docs/zoo/index.md [hunk 4]
```diff
@@ -5,32 +5,50 @@ These models can be used for prediction, feature extraction, and fine-tuning.
...
-| [BinaryDenseNet45](/zoo/api/#binarydensenet45)               | 64.59 %        | 85.21 %        | 13 939 240 | 7.54 MB  |
-| [BinaryDenseNet37Dilated](/zoo/api/#binarydensenet37dilated) | 64.34 %        | 85.15 %        | 8 734 120  | 5.25 MB  |
-| [BinaryDenseNet37](/zoo/api/#binarydensenet37)               | 62.89 %        | 84.19 %        | 8 734 120  | 5.25 MB  |
-| [BinaryDenseNet28](/zoo/api/#binarydensenet28)               | 60.91 %        | 82.83 %        | 5 150 504  | 4.12 MB  |
-| [BinaryResNetE18](/zoo/api/#binaryresnete18)                 | 58.32 %        | 80.79 %        | 11 699 368 | 4.03 MB  |
-| [Bi-Real Net](/zoo/api/#birealnet)                           | 57.47 %        | 79.84 %        | 11 699 112 | 4.03 MB  |
-| [DoReFaNet](/zoo/api/#dorefanet)                             | 53.39 %        | 76.50 %        | 62 403 912 | 22.84 MB |
-| [XNOR-Net](/zoo/api/#xnornet)                                | 44.96 %        | 69.18 %        | 62 396 768 | 22.81 MB |
-| [Binary AlexNet](/zoo/api/#binaryalexnet)                    | 36.30 %        | 61.53 %        | 61 859 192 | 7.49 MB  |
+The `literature` submodule contains the following models:
```
Comment (lgeiger) [inline docs/zoo/index.md]: Linking to the API docs would be nice

File: docs/zoo/index.md [hunk 5]
```diff
@@ -39,7 +57,7 @@ Weights can be downloaded automatically when instantiating a model. They are sto
 
 Larq Zoo ships with a command-line interface powered by [`zookeeper`](https://github.com/larq/zookeeper/), allowing you to reproduce the entire training process. If you want to improve an existing model or implement your own, we recommend to installing Larq Zoo in [development mode](https://github.com/larq/zoo/blob/master/CONTRIBUTING.md#project-setup).
 
-E.g. to reproduce the training of [Binary AlexNet](/zoo/api/#binaryalexnet) run:
+E.g. to reproduce the training of [Binary AlexNet](/zoo/api/literature/#binaryalexnet) run:
 
 ```shell
 lqz train binary_alexnet --dataset imagenet2012 --dataset-version 5.0.0
```
Comment (lgeiger) [inline docs/zoo/index.md]: I think this documentation needs some change as well.

**Pull request merged**